system/lib/pthread/library_pthread_stub.cpp

```diff
@@ -0,0 +1,31 @@
+#include <stdlib.h>
+#include <pthread.h>
+
+extern "C" {
+
+int pthread_key_create(pthread_key_t *key, void (*destructor)(void *))
+{
+	void **ptr = (void**)malloc(sizeof(void*));
+	*ptr = NULL;
+	*key = (pthread_key_t)ptr;
+	return 0;
+}
+
+int pthread_key_delete(pthread_key_t key)
+{
+	free((void*)key);
+	return 0;
+}
+
+void *pthread_getspecific(pthread_key_t key)
+{
+	return *(void**)key;
```

Comment:
the JS versions actually did work, while these are fake. that means code will regress? specifically code using these pthreads methods but not actually enabling pthreads. i think that includes python or something else in the test suite, that just uses pthreads for the key handling stuff. or am i missing something?